src/client/game/game.ts

```diff
@@ -290,12 +308,11 @@ export default class Game {
     // End server socket event handling
   }
 
-  private restart(token: string = "") {
+  private restart() {
```

Comment:
I realized we don't actually need to take the token here, because since we set it as a cookie it will be in the request by default